packages/components/src/components/hds/form/super-select/single/base.ts

```diff
@@ -45,9 +47,21 @@ export default class HdsFormSuperSelectSingleBase extends Component<HdsFormSuper
   }
 
   get resultCountMessage(): string {
+    if (typeof this.args.resultCountMessage === 'string') {
+      return this.args.resultCountMessage;
+    }
+
     return `${this.powerSelectAPI?.resultsCount || 0} total`;
   }
 
+  get resultCountMessageFunction() {
```

Comment:
```suggestion
  get resultCountMessageFunction(): PowerSelectSignature['Args']['resultCountMessage'] | undefined {
```
[Nit] Could add typing for the function since all other functions in the file are typed.